contents/blog/learnings-from-elon.md

```diff
@@ -0,0 +1,95 @@
+
+## Product drives marketing and sales
+
+There is a very product-first approach - many of his companies put something so cool on the internet, that it generates an extreme amount of word of mouth growth.
+
+We grow primarily through word of mouth. Long term, we think this is a winning strategy as it’s more efficient than a sales led model - you can reinvest everything back into the product, to build a bigger and bigger long term advantage.
+
+You can see this from the mind blowing specification of the Teslas, for example - they are incredibly powerful, superb value and very cool. There’s really no major compromise. Hence they rocketed in popularity.
+
+In the details but not the day to day running
+
+Elon attends very detailed design review sessions for many of the major projects he cares about. The book explains how he is regularly attending sessions to design a robot’s hands, for example. However, it goes on to describe him having 30+ direct reports. That is simply too many to be meaningful for daily progress.
+
+Tim and I are trying to work like this more and more - to go deep on the projects we can help out, and to “build the org” to cover the daily execution. Charles, in our exec team, makes sure our ops, marketing and customer success teams all function smoothly (each has a leader that reports to him), and we’re starting to do the same in engineering. Giving away some responsibility means you can add more value to the specific things you are uniquely good at (or which you love).
+
+## He’s erratic
+
+Elon has had a string of weird stuff on social media. Whilst this probably brings more awareness and a strange kind of marketing benefit, we think it’s a symptom of a very top down culture where things change based on the leader’s mood.
+
+We do not think this is a strength - perhaps it works in his companies because it creates a fear element, and more urgency as a result. 
+
+At PostHog, as mentioned, we’re more intrinsically motivated so ruling by fear would be harmful to performance.
+
+Whilst Tim and I technically are the CTO and CEO respectively, we more or less share the CEO role. We’ve been very effective like this. I think it’s because we’re a product led company, so what we ship has a very strong impact on marketing and sales (see “Product drives marketing and sales” above), so keeping a tight connection between these varied teams really matters.
+
+The key to making a split like this work is happily spending a lot of time together. This (and being ok sometimes missing out on decisions) is the only way to be decisive when two of you are paired like this. We talk a lot (several hours a week, minimum) - which can feel inefficient, but the reality is we need to optimize for organizational effectiveness, not our own schedules.
+
+## Likes the taste of his own blood
+
+Clickbait subtitle. You got me. Elon gives very direct feedback and makes painful decisions rapidly and doesn’t dwell on any of this stuff. For example, he 
```

Comment:
finish the sentence

Reply:
bought twitter? https://www.nbcnews.com/business/business-news/twitter-elon-musk-timeline-what-happened-so-far-rcna57532